icu4c/source/common/ucnv.cpp

```diff
@@ -323,7 +323,7 @@ ucnv_safeClone(const UConverter* cnv, void *stackBuffer, int32_t *pBufferSize, U
 U_CAPI UConverter* U_EXPORT2
 ucnv_clone(const UConverter* cnv, UErrorCode *status)
 {
-    return ucnv_safeClone(cnv, nullptr, nullptr, status);
+    return ucnv_clone(cnv, status);
```

Comment:
Were these changes needed to compile? I wonder if these need to be upstreamed? Then we can back this commit out and cleanly merge next upgrade.

Reply:
Correct. It has been a while since I upstreamed a change to ICU but I remember the maintainers were very fast and receptive. If we don't upstream this, we'll probably need to make these changes every time we upgrade so removing that would benefit our process and make upgrades more trivial.